Autopilot/AttitudeManager/nbproject/configurations.xml

```diff
@@ -193,8 +196,9 @@
         <property key="use-cci" value="false"/>
         <property key="use-iar" value="false"/>
       </C30>
-      <C30-AS>
-      </C30-AS>
+      <C30-AR>
+        <property key="additional-options-chop-files" value="false"/>
```

Comment:
Yeah I see what your saying. This file specifically has nothing to do with the new file additions though. The other config file does, which is the one that should be commited.